styles.css

```
@@ -0,0 +1,255 @@
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+
+/* --------------- 4.2 Introduction ------------------- */
+
+/* ------------------------------- 5. Media Queries -------------------------------- */
+
+button {
+    display: inline-block;
+    padding: 20px 50px;
+    font-size: 12px;
+    cursor: pointer;
+    text-align: center;
+    text-decoration: none;
+    outline: none;
+    color: white;
+    background-color: #ED4385;
+    border: none;
+    border-radius: 15px;
+    box-shadow: 0 5px #999;
+}
+button:active {
+    background-color: #3e8e41;
+    box-shadow: 0 5px #666;
+    transform: translateY(4px);
+}
+
+h1,h2,h3{
+    font-family: "Roboto Light", sans-serif;}
+
+p{
+    font-family:'Merriweather', sans-serif;}
+
+header{
+    background-color: #4845E4;
+    padding: 10px;
+}
```

Ik ga veel comments plaatsen met css wijzigingen. Ik hoop dat het een beetje duidelijk is. De bedoeling van al deze comments is dat het centreren van de content van de header wat netter is waardoor je bij het smaller maken van het venster alles netjes in het midden blijft staan.

Om te beginnen zou ik de header het volgende meegeven:
```suggestion
header{
    background-color: #4845E4;
    padding: 10px;
    display: flex;
    flex-direction: column;
    align-items: center;
}
```

Hiermee zeg je eigenlijk dat alle content met flex mag werken en dat alles onder elkaar komt te staan én in het midden 😛